### ISSUE TITLE
Audio out of sync when start time isn't zero

### ISSUE THREAD

**Issue by fireattack** [rigaya/NVEnc#386]
Basically the exact same issue as #322 but since the reporter closed it (due to them finding a workaround), I think I would open another one since the bug still exists.

The file (reported via `ffprobe`):
```
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'raw.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf59.17.101
  Duration: 00:00:35.12, start: 0.667000, bitrate: 9648 kb/s
  Stream #0:0[0x1](und): Video: h264 (High) (avc1 / 0x31637661), yuv420p(tv, bt709, progressive), 1920x1080 [SAR 1:1 DAR
 16:9], 9578 kb/s, 29.97 fps, 29.97 tbr, 16k tbn (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : [0][0][0][0]
  Stream #0:1[0x2](jpn): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 261 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]
```

I have tried to use either `--audio-copy` or `--audio-codec`, all three `--avsync` methods, 
```bash
nvencc64 -i raw.mp4 --avsync cfr --audio-codec aac --audio-bitrate 256 -o cfr_audio_encode.mp4
nvencc64 -i raw.mp4 --avsync cfr --audio-copy -o cfr_audio_copy.mp4
nvencc64 -i raw.mp4 --avsync forcecfr --audio-codec aac --audio-bitrate 256 -o forcecfr_audio_encode.mp4
nvencc64 -i raw.mp4 --avsync forcecfr --audio-copy -o forcecfr_audio_copy.mp4
nvencc64 -i raw.mp4 --avsync vfr --audio-codec aac --audio-bitrate 256 -o vfr_audio_encode.mp4
nvencc64 -i raw.mp4 --avsync vfr --audio-copy -o vfr_audio_copy.mp4
```
same result: the audio would be obviously off-sync.

NVENC info:
```
NVEncC (x64) 5.46 (r2126) by rigaya, Mar  5 2022 12:31:25 (VC 1929/Win)
  [NVENC API v11.1, CUDA 10.1]
 reader: raw, y4m, avi, avs, vpy, avsw, avhw [H.264/AVC, H.265/HEVC, MPEG2, VP8, VP9, VC-1, MPEG1, MPEG4, AV1]
```

I provide a sample input file here for investigation: [OneDrive](https://1drv.ms/v/s!Akq11jtCTJYwg4ppVa_AMJ_dUBklJw)

Thanks.

**Comment by rigaya** [comment]
Thank you for providing the sample.

It seems like there were trouble handling input files with audio and video having different timebase and start time isn't zero.

Shall be fixed with [NVEnc 6.00](https://github.com/rigaya/NVEnc/releases/tag/6.00).

**Comment by fireattack** [comment]
Thanks, verified it's fixed.
